### PR TITLE
Fix Seg Fault in Python 2.7 under 64bit OS

### PR DESCRIPTION
**Pull request by hewei-chn**

### CODE REVIEW

File: pcap.c [hunk 1]
```diff
@@ -278,7 +278,7 @@ pcap_PcapObject_read(PyObject *p)
       }
 
       if (len != sizeof(*hdr)) {
-        PyErr_Format(PyExc_IOError, "Read returned wrong number of bytes (%d)", len);
+        PyErr_Format(PyExc_IOError, "Read returned wrong number of bytes (%lu)", len);
```
Comment (vadmium) [inline pcap.c:281]: According to the Python documentation, it looks like this should actually be "%zd".

**Comment by vadmium** [comment]
Just so you know, I’m not the author of this, and only ever used the native Python pcap.py implementation, not the C implementation. But this looks like a sensible fix, so I’m happy to add it. Not sure if the original author (@nealy) is interested in maintaining this.

**Comment by vadmium** [comment]
I pushed your commit after changing the format string to "%zd".

**Comment by hewei-chn** [comment]
Thank you for your correction~

**Pull request closed without merging**